src/cf-api-controllers/controllers/build_controller.go

```diff
@@ -152,11 +154,16 @@ func (r *BuildReconciler) extractProcessTypes(build *buildv1alpha1.Build) (map[s
 
 	ret := make(map[string]string)
 	for _, process := range buildMetadata.Processes {
-		ret[process.Type] = process.Command
+		ret[process.Type] = extractFullCommand(process)
 	}
 	return ret, nil
 }
 
+func extractFullCommand(process launch.Process) string {
+	commandWithArgs := append([]string{process.Command}, process.Args...)
```

Comment:
I'm worried that this isn't a stable interface... It might've even been exclusively `Command` in the past and then it changed from under us...

Reply:
Yeah, it looks like it is actually a relatively recent change. Checkout this RFC:
https://github.com/buildpacks/rfcs/blob/main/text/0045-launcher-arguments.md#buildpacks

My hope is that the way I'm using `strings.Join` to construct this it will continue to support both cases. Looking forward to input from the kpack folks, though.

Reply:
👋  Hi from the kpack team. This looks good to me, and should be able to handle older and newer buildpacks. I don't foresee this interface changing any time soon.